tests/examples/opinions/united_states/virginislands_example.compare.json

```diff
@@ -7,6 +7,7 @@
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
     "docket_numbers": "SCT-CIV-2022-0046",
+    "judges": "Hodge, Rhys S.",
```

Comment:
and how is this a string and the other an array?

Reply:
Please see my answer  to your comment on nmariana_example.compare.json